mwdb/web/src/components/Profile/Views/ProfileCapabilities.js

```diff
@@ -37,6 +38,8 @@ function CapabilitiesTable({ profile }) {
 export default function ProfileCapabilities({ profile }) {
     return (
         <div className="container">
+            <h2>Capabilities</h2>
+            <p className="lead">Here is the list of your superpowers:</p>
```

Comment:
I decided to remove that kind of headers from linked components with Settings and Profile, because we have breadcrump but it nice look here, so only change single word to allow reuse that component for other accounts.
```suggestion
            <h2>Capabilities</h2>
            <p className="lead">Here is the list of account superpowers:</p>
```